Makefile

```diff
@@ -14,30 +16,40 @@ build/aws-stack.json: $(wildcard templates/*.yml)
 	bundle exec cfoo $^ > $@
 
 setup:
-	which bundle || gem install bundler --no-ri --no-rdoc
-	bundle install --path vendor/bundle
+	bundle check || ((which bundle || gem install bundler --no-ri --no-rdoc) && bundle install --path vendor/bundle)
+	cp config.json.example config.json
```

Comment:
Yeah, I think you're right. A nice "run this command and edit your conf!" message is a better idea. I'll patch that.